packages/dataclass-editor/src/master/DataClassMasterContent.tsx

```diff
@@ -97,16 +109,53 @@ export const DataClassMasterContent = () => {
   useUpdateSelection(table);
 
   const deleteField = () => {
-    const { newData: newFields, selection } = deleteFirstSelectedRow(table, dataClass.fields);
+    const { newData: newFields, selection } = deleteAllSelectedRows(table, dataClass.fields);
     const newDataClass = structuredClone(dataClass);
     newDataClass.fields = newFields;
     setDataClass(newDataClass);
     setSelectedField(selection);
   };
 
+  const updateOrder = (moveId: string, targetId: string) => {
+    const selectedRows = table.getSelectedRowModel().flatRows.map(r => r.original.name);
```

Comment:
Is there a reason to use `flatRows` instead of `rows` when there are no subRows?

Reply:
I think in this case the api's are the same.